aws/flatcar-linux/kubernetes/ami.tf

```diff
@@ -1,13 +1,15 @@
 locals {
-  # Pick a CoreOS Container Linux derivative
-  ami_id = local.flavor == "flatcar" ? data.aws_ami.flatcar.image_id : data.aws_ami.coreos.image_id
+  # Pick a Container Linux derivative
+  ami_id = local.flavor == ${element(concat(data.aws_ami.flatcar.*.image_id, data.aws_ami.coreos.*.image_id, list("")), 0)}
```

Comment:
`local.flavor == ` part should be dropped here? Currently `ami_id` will be bool if I understand the code correctly.

Reply:
Yes, my mistake. I'm fixing it and I will test again.